src/main/java/com/laytonsmith/core/functions/InventoryManagement.java

```diff
@@ -1812,6 +1812,88 @@ public CHVersion since() {
 			return CHVersion.V3_3_1;
 		}
 	}
+	
+	@api(environments = {CommandHelperEnvironment.class})
+	public static class pset_selected_quickbar_slot extends AbstractFunction {
+
+		@Override
+		public String getName() {
+			return "pset_selected_quickbar_slot";
+		}
+
+		@Override
+		public Integer[] numArgs() {
+			return new Integer[]{1};
+		}
+
+		@Override
+		public String docs() {
+			return "void {[player], slotNumber} Sets the selected quickbar slot of the given or executing player"
+					+ " to the given slot. The slots are in range of [1-9].";
```

Comment:
I did this because the slot numbers now correspond to the keyboard key you have to press to select it. I thought that was nice, but I can change it if desired.